.configure

```diff
@@ -5,17 +5,17 @@
   "files_to_copy": [
     {
       "file": "shared/google_cloud_keys.json",
-      "destination": ".configure-files/google_cloud_keys.json",
+      "destination": "~/.configure/wordpress-ios/secrets/google_cloud_keys.json",
       "encrypt": true
     },
     {
       "file": "iOS/WPiOS/project.env",
-      "destination": ".configure-files/project.env",
+      "destination": "~/.configure/wordpress-ios/secrets/project.env",
       "encrypt": true
     },
     {
       "file": "iOS/app_store_connect_fastlane_api_key.json",
-      "destination": ".configure-files/app_store_connect_fastlane_api_key.json",
+      "destination": "~/.configure/wordpress-ios/secrets/app_store_connect_fastlane_api_key.json",
       "encrypt": true
```

Comment:
I wonder how we could nudge people into cleaning up the old locations – i.e. running `git clean` or `rm .configure-files/*` – after this migration 🤔 
(Or does `configure_apply` takes care of cleaning up this dir before re-applying the config? Though that means the action would have to know about this path being a legacy default path being used…)

Reply:
Good point 🤔 

There are a few files still to move (see #16806) once that's done, `.configure-files` would become a legacy location.

I wonder if we should do something a bit dirty and make `configure` aware of whether it's running inside one of the Apps division projects and take care of running `git clean` or suggesting to do so to the user if it detects that all the files in `.configure` are pointing to the new `~/.configure` home.

I say that's dirty because it adds Automattic/Apps specific information to the tool and that's undesirable from an OSS point of view. Pragmatically speaking, though, that might be an acceptable tradeoff. The tool is open-source but it's so backed into Automattic specific workflows (even just the fact that it's part of the release toolkit instead of being standalone) that I highly doubt anyone other than us is using it right now or would consider doing so. Another factor in favor of doing this is that we're talking about removing secrets for security reasons.

Reply:
What about adding support in the tool for an entry in the `.configure` file specifying paths to clean? That way we wouldn't hardcode in the tool the `.configure-files` path to be cleaned, but make it configurable, and thus avoid the issue in making it too a8c-specific/aware. Best of both worlds, right?